driver/fixtures/fixtures_test.go

```diff
@@ -16,19 +17,39 @@ var Suite = &fixtures.Suite{
 	Ext:  ".py",
 	Path: filepath.Join(projectRoot, fixtures.Dir),
 	NewDriver: func() driver.BaseDriver {
-		return driver.NewExecDriverAt(filepath.Join(projectRoot, "build/bin/native"))
+		return driver.NewExecDriverAt(filepath.Join(projectRoot,
+			"build/.local/bin/python_driver"))
```

Comment:
Ideally, we shouldn't change the path to native driver (`../../build/bin/native`). That script should use relative paths so it doesn't matter if it's in `/opt/bblfsh/driver` or in the project directory.